py-polars/polars/functions/lazy.py

```diff
@@ -499,7 +720,8 @@ def tail(column: str, n: int = 10) -> Expr:
     └─────┘
 
     """
-    return F.col(column).tail(n)
+    col = combine_column_names(column)
```

Comment:
I don't see the point of this one?

Reply:
Looks like a leftover/remnant from the `col` conversion; not necessary any more.